docs/user_guide/mitigation.rst

```diff
@@ -651,7 +1365,23 @@ Grid Search
 
 .. _references:
 
+   .. [#1] Agarwal, Dudik, Wu `"Fair Regression: Quantitative Definitions and
```

Comment:
We have recently moved to BibTex! Please add the references in the `ref.bib` file and cite in the text accordingly. See also #1056 and #1069